sample/src/androidTest/java/com/schibsted/spain/barista/sample/AssertionsTest.java

```diff
@@ -70,6 +75,11 @@ public void checkExpectedText() throws Exception {
     assertDisplayed(R.id.visible_view, "Hello world!");
   }
 
+  @Test
+  public void checkExpectedTextContains() throws Exception {
```

Comment:
This name doesn't sound very clear.
How about `checkContainsText` o `checkPartiallyContainsText`?

Reply:
Done